### ISSUE TITLE
模型输出和优化损失函数

### ISSUE THREAD

**Issue by xmser** [QiShanZhang/SLSforASVspoof-2021-DF#7]
感谢你音频伪造的方法思路。我在尝试训练和推理预测时，有一个疑惑。请问model.py 93行这里对logits使用了logsoftmax

![Image](https://github.com/user-attachments/assets/1b8601e4-c75e-44c5-9f82-a52837de1a2d)

那么为什么训练的时候使用的是crossentropy，而不是nllloss。我的一般理解是crossentropy+logits，nllloss+logsoftmax。

![Image](https://github.com/user-attachments/assets/270f1cb9-3363-4a11-b752-32e5dde48128)

然后推理阶段，我的目标是0/1二分类该音频，我是否可以直接使用softmax函数，根据我的实际操作，在pretrained模型上，是可行的。
如果我要train from scratch，我是否可以按照现在的架构和training策略使用？

再次感谢～
